README.md

```diff
@@ -1,47 +1,54 @@
 # IT-Projekt: Schlaue Lise
 
-# Anaconda Installation
+## Inhaltsverzeichnis
+
+1. [Anaconda Installation](#anaconda-installation)
+2. [Rasa X](#rasa-x)
+3. [CI / CD Pipeline](#ci--cd-pipeline)
+4. [Probleme mit Timeout-Error](#probleme-mit-timeout-error)
+5. [Rasa Befehle](#rasa-befehle)
+
+## Anaconda Installation
 
 Eine benutzerfreundliche Python-Distribution ist Anaconda.
```

Comment:
```suggestion
Anaconda ist eine benutzerfreundliche Python-Distribution.
```

Reply:
klingt für mich weniger umständlich

Reply:
jo

Reply:
Das is übrigens doppelt, eins davon kann weg

Reply:
Bin dafür, dass die komplette Line gelöscht wird.
```suggestion
```

Reply:
Hab die Zeile komplett gelöscht.